geoengine/workflow.py

```diff
@@ -425,6 +425,39 @@ def get_provenance(self) -> List[ProvenanceOutput]:
 
         return [ProvenanceOutput.from_response(item) for item in response]
 
+    def save_as_layer(self, name: str, description: str, bbox: QueryRectangle) -> InternalDatasetId:
```

Comment:
maybe change order and give description a default value of empty string?

Reply:
also we should name the method `save_as_dataset` or just `save`

Reply:
a2b883c